lib/nile/src/Nile.ts

```diff
@@ -13,18 +14,62 @@ import {
   ConfigurationParameters,
 } from './generated/openapi/src/runtime';
 
+export class EventsApi {
+  static onCounter: number = 0;
+  entities: EntitiesApi;
+  timers: { [key: string]: NodeJS.Timer };
+
+  constructor(entities: EntitiesApi) {
+    this.entities = entities;
+    this.timers = {};
+  }
+
+  on(options: {type: string, seq: any}, listener: (event: InstanceEvent) => Promise<void>): number  {
+    const id = EventsApi.onCounter++;
+    var seq = options.seq;
+    const getEvents = async () => {
+      console.log('calling with seq:', seq)
```

Comment:
possibly could add a `debug` option. I know since this is primarily a CLI thing, could be good to have a cli wrapper to pass a flag to log info like this out.